docker-stack-wait.sh

```diff
@@ -8,6 +8,7 @@ set -e
 trap "{ exit 1; }" TERM INT
 opt_h=0
 opt_r=0
+opt_v=0
```

Comment:
For consistency, the `-v` should be after the `-t` option throughout the file (alphabetical).